<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WelcomeWizard</class>
 <widget class="QWizard" name="WelcomeWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Welcome</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QWizardPage" name="wizardPage1">
   <property name="title">
    <string>Welcome</string>
   </property>
   <property name="subTitle">
    <string>This wizard will guide you through the initial configuration.</string>
   </property>
  </widget>
  <widget class="QWizardPage" name="wizardPage2">
   <property name="title">
    <string>ISO paths</string>
   </property>
   <property name="subTitle">
    <string>If you have a collection of game images, you can add them to the path list here. Gekko will automatically show a list of your collection on startup then.</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="edit_path">
        <property name="text">
         <string/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>Select a path to add ...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_browse_path">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="GPathList" name="path_list"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="button_add_path">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GPathList</class>
   <extends>QListWidget</extends>
   <header>path_list.hxx</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
